packages/pangea-node-sdk/src/types.ts

```diff
@@ -2268,50 +2441,105 @@ export namespace Share {
   }
 
   export interface ListFilter {
+    /**
+     * Only records where the object exists in the supplied parent folder path
+     * name.
+     */
     folder?: string;
     folder__contains?: string[];
     folder__in?: string[];
```

Comment:
It seems they are not in the docs neither. We should remove `folder__contains` and `folder__in`

Reply:
Alright, done.